admin/resources/js/stories/InputError.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Story, Meta } from "@storybook/react";
+import InputError from "../components/form/InputError";
+
+const meta: Meta = {
+  title: "Input Error",
```

Comment:
might this be better named `Components/Input Error` for maintaining a [hierarchy](https://storybook.js.org/docs/riot/writing-stories/naming-components-and-hierarchy)? Or `Hydrogen/Input Error` and then changing Input Label to match?

Reply:
I'll change this to match InputLabel